webpack/assets/javascripts/react_app/components/Bookmarks/Bookmarks.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import EllipisWithTooltip from 'react-ellipsis-with-tooltip';
 import { Dropdown, MenuItem, Spinner, Icon } from 'patternfly-react';
-import SearchModal from './components/SearchModal';
+import SearchModal from './components/SearchModal/index';
```

Comment:
This will lead to the same place as before the change

Reply:
@MariaAga I tried changing it back but it seems the change was needed